fossunited/stack/utils.py

```diff
@@ -0,0 +1,15 @@
+import frappe
+
+
+def get_stack_dict():
+    stacks = frappe.get_all(doctype="Stack", fields=["name1", "icon", "category"], page_length=999)
+
+    stack_dict = {}
+
+    for item in stacks:
+        if item.category not in stack_dict:
+            stack_dict.update({item.category: []})
+
+        stack_dict[item.category].append(item)
```

Comment:
why not use a [`collections.defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict) here? please see https://docs.python.org/3/library/collections.html#defaultdict-examples for additional information